<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>397</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export to...</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="outputFormatLabel">
     <property name="text">
      <string>Output Format</string>
     </property>
     <property name="buddy">
      <cstring>outputFormatComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QComboBox" name="outputFormatComboBox">
     <item>
      <property name="text">
       <string>PDF</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PNG</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="widthLabel">
     <property name="text">
      <string>Width</string>
     </property>
     <property name="buddy">
      <cstring>widthSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QDoubleSpinBox" name="widthSpinBox">
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>400.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="heightLabel">
     <property name="text">
      <string>Height</string>
     </property>
     <property name="buddy">
      <cstring>heightSpinBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDoubleSpinBox" name="heightSpinBox">
     <property name="minimum">
      <double>0.010000000000000</double>
     </property>
     <property name="maximum">
      <double>400.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QComboBox" name="measureComboBox">
     <item>
      <property name="text">
       <string>cm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>in</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="resLabel">
     <property name="text">
      <string>Resolution</string>
     </property>
     <property name="buddy">
      <cstring>resHSlider</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="3">
    <widget class="QSlider" name="resHSlider">
     <property name="minimum">
      <number>30</number>
     </property>
     <property name="maximum">
      <number>1200</number>
     </property>
     <property name="value">
      <number>72</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QSpinBox" name="resSpinBox">
     <property name="minimum">
      <number>30</number>
     </property>
     <property name="maximum">
      <number>1200</number>
     </property>
     <property name="value">
      <number>72</number>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QComboBox" name="resRatioComboBox">
     <item>
      <property name="text">
       <string>px / in</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>px / cm</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0" colspan="7">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>outputFormatComboBox</tabstop>
  <tabstop>widthSpinBox</tabstop>
  <tabstop>heightSpinBox</tabstop>
  <tabstop>measureComboBox</tabstop>
  <tabstop>resHSlider</tabstop>
  <tabstop>resSpinBox</tabstop>
  <tabstop>resRatioComboBox</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ExportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ExportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>190</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resHSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>resSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>resHSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
